develop

```diff
@@ -0,0 +1,14 @@
+package com.itschool.course;
+
+public class CursMain4 {
```

Comment:
I understand this file has been deleted, but I don't understand why is it shown green here. Most probably a GitHub glitch.